src/notation/internal/masternotation.cpp

```diff
@@ -333,53 +333,6 @@ void MasterNotation::applyOptions(mu::engraving::MasterScore* score, const Score
 
         QString text("<sym>metNoteQuarterUp</sym> = %1");
         double bpm = scoreOptions.tempo.valueBpm;
-        switch (ts.denominator()) {
-        case 1:
-            bpm /= 4;
-            break;
-        case 2:
-            bpm /= 2;
-            break;
-        case 4:
-            break;
-        case 8:
-            if (ts.numerator() % 3 == 0) {
-                bpm /= 1.5;
-            } else {
-                bpm *= 2;
-            }
-            break;
-        case 16:
-            if (ts.numerator() % 3 == 0) {
-                bpm *= 1.5;
-            } else {
-                bpm *= 4;
-            }
-            break;
-        case 32:
-            if (ts.numerator() % 3 == 0) {
-                bpm *= 3;
-            } else {
-                bpm *= 8;
-            }
-            break;
-        case 64:
-            if (ts.numerator() % 3 == 0) {
-                bpm *= 6;
-            } else {
-                bpm *= 16;
-            }
-            break;
-        case 128:
-            if (ts.numerator() % 3 == 0) {
-                bpm *= 6;
-            } else {
-                bpm *= 16;
-            }
-            break;
-        default:
-            break;
-        }
```

Comment:
I can't find or think of a single case where this makes sense...

Reply:
My guess is perhaps the original intention was something to do with this notion of [ticksPerBeat](https://github.com/musescore/MuseScore/blob/9daf58b89b9edcac9a15098fb37921f44a159220/src/notation/internal/notationinteraction.cpp#L4400)